pkgs/applications/version-management/git-lfs/default.nix

```diff
@@ -1,43 +1,56 @@
-{ lib, buildGoPackage, fetchFromGitHub, ronn, installShellFiles }:
+{ lib, buildGoModule, fetchFromGitHub, ronn, installShellFiles, git, testers, git-lfs }:
 
-buildGoPackage rec {
+buildGoModule rec {
   pname = "git-lfs";
   version = "3.2.0";
 
   src = fetchFromGitHub {
-    rev = "v${version}";
     owner = "git-lfs";
     repo = "git-lfs";
+    rev = "v${version}";
     sha256 = "sha256-3gVUPfZs5GViEA3D7Zm5NdxhuEz9DhwPLoQqHFdGCrI=";
+    leaveDotGit = true;
   };
 
-  goPackagePath = "github.com/git-lfs/git-lfs";
+  vendorSha256 = null;
 
   nativeBuildInputs = [ ronn installShellFiles ];
 
-  ldflags = [ "-s" "-w" "-X ${goPackagePath}/config.Vendor=${version}" "-X ${goPackagePath}/config.GitCommit=${src.rev}" ];
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/git-lfs/git-lfs/v${lib.versions.major version}/config.Vendor=${version}"
+  ];
 
   subPackages = [ "." ];
 
   preBuild = ''
-    pushd go/src/github.com/git-lfs/git-lfs
-      go generate ./commands
-    popd
+    go generate ./commands
   '';
```

Comment:
Please remove this. IIUC is not necessary, it regenerates the .ronn documents, and it should be handled upstream.

Reply:
I'm so sorry, my bad.
 This command is necessary for injecting help information into the commands. The same it's applied in ArchLinux https://github.com/archlinux/svntogit-community/commit/476c403f4cbfcb29e84a47130cef1d4797a09057. 
I'm reverting it to your original implementation. Thanks!